generators/cmd/sigs/sig-list.gomd

```diff
@@ -19,7 +19,7 @@
     */
 -}}
 {{- /* gotype: kubevirt.io/community/generators/cmd/sigs.SigListTemplateData */ -}}
-{{ define "group" }}|[{{ .Name }}]({{ .Dir }}/charter.md) |[{{ .Label }}](https://github.com/kubevirt/kubevirt/labels/{{ .Label }})|{{ if .Leadership }}<ol>{{ range $index2, $chair := .Leadership.Chairs }}<li>[{{ $chair.Name }}](https://github.com/{{ $chair.Github }}), {{ $chair.Company }}</li>{{ end }}</ol>{{ end }} |{{ if .Contact }}[Slack]({{ .Contact.Slack}})<br/> [Mailing List]({{ .Contact.MailingList }}){{ end }} |<ul>{{ range $index3, $meeting := .Meetings }}<li>{{ $meeting.Description }}: [ {{ $meeting.Day }} at {{ $meeting.Time }} {{ $meeting.TZ }} ({{ $meeting.Frequency }}) ]({{ $meeting.URL }})</li>{{ end }}</ul> |{{ end }}
```

Comment:
@jean-edouard so this is the change in the template that I promised, hope it's ok for you to do this here - since we are changing the template anyhow I think that's fair.